.credo.exs

```diff
@@ -0,0 +1,18 @@
+%{
+  configs: [
+    %{
+      name: "default",
+      files: %{
+        included: ["lib/", "test/"]
+      },
+      strict: true,
+      color: true,
+      checks: %{
+        disabled: [
```

Comment:
I always though that you should explicitly enable Credo checks in the `checks.enabled` list.
Does Credo run all checks even if you don't mention them?

If you need a reference, you can always start by generating the default Credo config:

```sh
mix credo gen.config
```

Reply:
You're right.
If we defined the `enabled` list, then *only* those checks would be run.
So in your case it's easier to mention the TODO check within the `disabled` list, since mentioning it as
```elixir
enabled: [
  {Credo.Check.Design.TagTODO, [exit_status: 0]}
]
```
would enable *only* the TODO check.